src/components/20-molecules/dropdown/index.js

```diff
@@ -251,14 +256,62 @@ class AXADropdown extends NoShadowDOM {
         // case 2: already open, navigate
         this.navigateByKeyboard(arrowKey);
       }
+    } else {
+      // selection of first matching dropdown item by quickly typing a prefix
+      // of the item's 'name' value, e.g. 'afgh' for selecting Afghanistan in a country list
+      this.handleAutosuggest(key);
+    }
+  }
+
+  handleAutosuggest(key) {
+    // ready for autosuggestions, by having focus on the non-native UI?
+    if (document.activeElement !== this.button) {
```

Comment:
Is this just a safety measure, or are there actual cases where this is not true, when this method is called?

Reply:
Safety measure, defensive programming